test/twoDo/services/UserTaskServiceTest.java

```diff
@@ -15,7 +16,8 @@ public UserTaskServiceTest(String method)
 	@Override
 	public void setUp()
 	{
-		service = new UserTaskService();
+		service = new UserTaskService(
```

Comment:
wow three places had to change. Let's put the service in the applicationWrapper.